appcenter-data/Data.js

```diff
@@ -69,7 +69,7 @@ Data.WriteOptions = class {
 };
 
 function convertTimestampToDate(result) {
-    if (!result === undefined || !result.lastUpdatedDate === undefined) {
+    if (result === undefined || result.lastUpdatedDate === undefined) {
```

Comment:
Doesn't handle `null` case, if that needs handling.

Reply:
Would `!result` take care of both?

Reply:
Yep, `(!result || !result.lastUpdatedDate)`

Reply:
Awesome, added